src/lib/messenger.js

```diff
@@ -34,7 +34,8 @@ export default function messenger(bot) {
       } else if (inputParser.isAskingForBoegivar(text)) {
         handleBoegivar(bot, message.chat, user.firstName);
       } else if (inputParser.isAskingForTourettes(text)) {
-        handleTourettes(bot, message.chat, text);
+        const commandText = text.substr(11, text.length);
```

Comment:
Ah! I understand the principle and advantages, I just wanted to reduce the amount of calls, will try to  remember henceforth.
I'll clean it and remove the other returns etc.